README.md

```diff
@@ -1,26 +1,12 @@
 # nextcloud-neon
 
-A beautiful convergent cross-platform client for Nextcloud written in Flutter.
+<img src="assets/logo.svg" alt="Neon logo" width="200"/>
 
-See [here](packages/app/README.md) for screenshots and other material regarding the app.
-
-This repository not only contains the Neon app, but also a Nextcloud client written in Dart.  
-The client will replace https://github.com/provokateurin/dart-nextcloud which is an older unmaintained client I wrote some time ago.  
-
-The development of this app and client just started, there will be a lot of changes and new features coming soon.
-
-Additional documentation is very much appreciated. If you find something that you think should be documented, please open an issue or pull request.
-
-## Development and support
-
-Either open an issue if you exactly know what you need or your problem is, otherwise please ask in Matrix.
-The Matrix space is here: https://matrix.to/#/#nextcloud-neon:matrix.org.  
-There are separate rooms for the client and the app. If you don't know which channel you should use if you have a question, just use the app channel.
+A framework for building convergent cross-platform Nextcloud clients using Flutter.
 
 ## Features
```

Comment:
This section could look a bit nicer, but I have no idea what that could be. I was thinking maybe adding the app icons somewhere could look nice.